Unix/configure

```diff
@@ -2231,20 +2330,41 @@ echo "created $fn"
 
 fn=$configuredir/GNUmakefile
 
-cat > $fn <<EOF
+if [ "$disable_makefile_gen" != "1" ]; then
+    cat > $fn <<EOF
 .PHONY: check
 .PHONY: tests
 
 __ROOT=$root
 export OUTPUTDIR=$outputdir
-export SYSCONFDIR=$sysconfdir
-export DATADIR=$datadir
 export ROOT=$root
+export ENABLE_ULINUX=$enable_ulinux
+
+OMI_CONFIGURE_QUALS=--enable-microsoft --disable-makefile-gen
 
 all:
+ifeq (\$(ENABLE_ULINUX),1)
+
+	@echo '========================= Performing Building OMI (0.9.8)'
```

Comment:
Okay, changed to `Performing Building OMI (SSL 0.9.8)`. Similar change for SSL 1.0.0.